src/v2i-hub/CDASimAdapter/test/sensors.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "id": "SomeID",
```

Comment:
Can we change this to be sensorId. I think id conflicts with Interaction (MOSIAC RTI) Id field.

Reply:
Addressed

Reply:
sensorId instead of sensor_id to be consistant with naming convention.

Reply:
updated